lua/telescope/builtin/internal.lua

```diff
@@ -637,9 +648,20 @@ internal.buffers = function(opts)
       results = buffers,
       entry_maker = opts.entry_maker or make_entry.gen_from_buffer(opts),
     },
-    previewer = conf.grep_previewer(opts),
+    previewer = previewers.buffers.new(opts),
     sorter = conf.generic_sorter(opts),
     default_selection_index = default_selection_idx,
+    attach_mappings = function(_, _)
+      action_set.select:enhance {
+        post = function()
+          local entry = action_state.get_selected_entry()
+          for opt, value in pairs(winopts) do
+            vim.api.nvim_win_set_option(0, opt, value)
+          end
+          vim.api.nvim_win_set_cursor(0, { entry.lnum, entry.col })
```

Comment:
this will break if `previewer = false` is set. i think

And then we dont actually need any to do anything

Reply:
Yeah, I've refactored it to use `previewer.setup` and `previewer.teardown` already. See the other comment whether you are fine with this.

Reply:
Yeah its still broken if `previewer = false`. 
`E5108: Error executing lua ...lugged/telescope.nvim/lua/telescope/builtin/internal.lua:647: Argument "pos" must be a [row, col] array`

We do not have `col` by default just do `entry.col or 1`.Or dont enhance the action if `opts.previewer = false`

Reply:
Yeah, that was just a stupid oversight on my part, sorry; I've burnt a bit on this PR and overread in coop with the other issue :sweat_smile: 

Will fix later.

Reply:
`column` in `nvim_win_set_cursor` is zero-indexed, so `entry.col or 0`; I also mix up zero and one indexing across functions more often than I'd like :laughing: